perses/charts/templates/alerts.yaml

```
@@ -0,0 +1,14 @@
+{{ if .Values.perses.serviceMonitor.selfMonitor }}
+{{ $root := . }}
+{{- range $path, $bytes := .Files.Glob "alerts/*.yaml" }}
+{{ $fileName := base $path | trimSuffix ".yaml" }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "release.name" $root }}-{{ $fileName }}-alerts
+  labels:
+    {{- include "perses.labels" $root | nindent 4 }}
+spec:
+{{ tpl (printf "%s" $bytes) $root | indent 2 }}
```

in case we add stuff to alerts folder
```suggestion
---
```